appyters/Independent_Enrichment_Analysis/Independent_Enrichment_Analysis.ipynb

```diff
@@ -223,10 +418,17 @@
     "set_filename = {{ set_kind.value[0] }}\n",
     "{%- endif %}\n",
     "\n",
-    "\n",
-    "library_filename = \"{{library_filename.value}}\"\n",
+    "library_kind = \"{{ library_kind.raw_value }}\"\n",
```

Comment:
please put this in your if statement as
```
{% if library_kind.raw_value == 'Upload' %}
library_kind = 'Upload'
```

and never directly write `{{ anything.raw_value }}` -- this is because raw_value is unconstrained and could contain arbitrary code.